solutions/arrays/rotate-matrix.js

```diff
@@ -0,0 +1,39 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+function rotate(arr, numOfRows){
+	console.log(arr[0].length);

```

Comment:
@DevratAgarwal Please do not print inside this function. Read about single responsibility principle. Perhaps we can avoid writing a function.